Sources/ApodiniMigratorShared/EncodeDecode/DecodingStrategies.swift

```diff
@@ -87,3 +87,33 @@ public extension JSONDecoder {
         return self
     }
 }
+
+extension JSONDecoder.DateDecodingStrategy: Equatable {
```

Comment:
As far as I can see you have moved the extension from the test target to the main target. Do we need to have them in the main target? If so, would it even make sense to also move these extensions to `ApodiniDocumentExport`? The same also applies to the EncodingStrategies.

Reply:
Good catch! 💪🏼 Looks to me like we do not need this in the main target so it makes sense to keep this in the test to make it clear.